.github/workflows/release-workflow.yml

```
@@ -0,0 +1,72 @@
+name: Build and Push Image
+on:
+  push:
+    branches:
+      - main
```

The `main` branch for this repo should be `master`.

[At least what's I'm getting from here](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-branches)

```suggestion
      - master
```

---

Should we build and push the "wip" images on every commit so we can test them out?

---

My idea was to add a test deployment (I was thinking about 'kind' action) to see if everything is working, but it will require a bit of time and planning.
For now, what I did up to now was to checkout the PR branches, build a test image, run a consistency check and then run the build and publish process on merge.

You triggered a lot of ideas, I need some time to capitalize on them and end up with a proposal!